R/geotargets-option.R

```diff
@@ -0,0 +1,65 @@
+#' Get or Set geotargets Options
+#'
+#' Get or set behavior for geospatial data target stores using geotargets-specific global options.
+#'
+#' @param option_name Character. Option name. See Details.
+#'
+#' @details
+#'
+#' ## Available Options
+#'
+#'  - `"geotargets.gdal.raster.creation_options"` - set the GDAL creation options used when writing raster files to target store (default: `"ENCODING=UTF-8"`)
```

Comment:
Can one supply multiple options?  If so, how should they be delimited?

Reply:
Great point, but I think this might be better as a separate issue: https://github.com/njtierney/geotargets/issues/26